common/src/types/v0/store/node.rs

```diff
@@ -73,25 +79,49 @@ impl NodeSpec {
     pub fn cordon(&mut self, label: String) {
         self.cordon_labels.push(label);
     }
+    /// Drain node by applying the drain label.
+    pub fn drain(&mut self, label: String) {
+        self.drain_labels.push(label);
+    }
     /// Uncordon node by removing the corresponding label.
     pub fn uncordon(&mut self, label: String) {
         if let Some(index) = self.cordon_labels.iter().position(|l| l == &label) {
             self.cordon_labels.remove(index);
         }
+        if let Some(index) = self.drain_labels.iter().position(|l| l == &label) {
+            self.drain_labels.remove(index);
+        }
     }
     /// Returns whether or not the node is cordoned.
     pub fn cordoned(&self) -> bool {
-        !self.cordon_labels.is_empty()
+        !self.cordon_labels.is_empty() || !self.drain_labels.is_empty()
```

Comment:
How do we expose these 2 label fields to the user?
A user cancels a drain by uncordoning right?

Reply:
kubectl-plugin get node <node id> --show-drain
 lists the drain labels

    kubectl-plugin get node <node id> --show-cordon-labels
lists both sets of labels in a single comma-separated list

```
./kubectl-plugin/bin/kubectl-mayastor get node node-1-10196 --show-cordon-labels
 ID            GRPC ENDPOINT         STATUS  CORDONED  CORDON LABELS 
 node-1-10196  157.90.227.228:10124  Online  true      cordon1, drain1 
```
```
./kubectl-plugin/bin/kubectl-mayastor get node node-1-10196 --show-drain
 ID            GRPC ENDPOINT         STATUS  CORDONED  DRAIN STATE  DRAIN LABELS 
 node-1-10196  157.90.227.228:10124  Online  true      Drained      drain1
 ```
 Cancelling a drain is done with the uncordon command